### PR TITLE
feat: UI update "Exchange tokens" popup

### PR DESCRIPTION
**Pull request by aineo**
- chore: update buy and sell popup
- feat: add links to exchange aggregators

### CODE REVIEW

**Comment by aineo** [comment]
[UI: Update "Exchange tokens" popup (copied from iOS)](https://trello.com/c/I3DPzscQ/587-ui-update-exchange-tokens-popup-copied-from-ios)

File: src/locales/zh.json [hunk 1]
```diff
@@ -104,15 +104,16 @@
...
     "buy_tokens_anonymously": "在网站上匿名",
+    "buy_tokens_btn": "购买和出售硬币",
+    "buy_tokens_btn_link": "https://adamant.im/buy-tokens/",
     "buy_tokens_exchanger": "聊天内交换器",
     "copied": "已复制",
     "copy": "复制",
     "copy_address": "复制地址",
     "copy_uri": "复制共享链接",
     "create_qrcode": "创建二维码",
+    "exchanges_on": "{ aggregator } 上的交易所",
```
Comment (bludnic) [inline src/locales/zh.json:115]:
```suggestion
    "exchanges_on": "{aggregator} 上的交易所",
```

File: src/locales/ru.json [hunk 2]
```diff
@@ -104,15 +104,16 @@
...
     "buy_tokens_anonymously": "Анонимно на сайте",
+    "buy_tokens_btn": "Купить или продать монеты",
+    "buy_tokens_btn_link": "https://adamant.im/ru-buy-tokens/",
     "buy_tokens_exchanger": "Обменник в чате",
     "copied": "Скопировал",
     "copy": "Копировать",
     "copy_address": "Копировать адрес",
     "copy_uri": "Копировать ссылку",
     "create_qrcode": "Создать QR-код",
+    "exchanges_on": "Биржи на { aggregator }",
```
Comment (bludnic) [inline src/locales/ru.json:115]:
```suggestion
    "exchanges_on": "Биржи на {aggregator}",
```

File: src/locales/en.json [hunk 3]
```diff
@@ -104,15 +104,16 @@
...
     "buy_tokens_anonymously": "Anonymously on website",
+    "buy_tokens_btn": "Buy & sell coins",
+    "buy_tokens_btn_link": "https://adamant.im/buy-tokens/",
     "buy_tokens_exchanger": "In-chat Exchanger",
     "copied": "Copied",
     "copy": "Copy",
     "copy_address": "Copy address",
     "copy_uri": "Copy Share link",
     "create_qrcode": "Create a QR code",
+    "exchanges_on": "Exchanges on { aggregator }",
```
Comment (bludnic) [inline src/locales/en.json:115]:
```suggestion
    "exchanges_on": "Exchanges on {aggregator}",
```

File: src/locales/de.json [hunk 4]
```diff
@@ -73,13 +73,15 @@
...
   "home": {
     "balance": "Balance",
+    "buy_tokens_btn": "Buy & sell coins",
+    "buy_tokens_btn_link": "https://adamant.im/de-buy-tokens/",
     "copied": "Kopiert",
     "copy": "Copy",
     "copy_address": "Copy address",
     "copy_uri": "Copy Share link",
     "create_qrcode": "Create QR code",
+    "exchanges_on": "Exchanges on { aggregator }",
```
Comment (bludnic) [inline src/locales/de.json:83]:
```suggestion
    "exchanges_on": "Exchanges on {aggregator}",
```

**Review by bludnic — APPROVED** [review]

**Pull request merged**